app/uk/gov/hmrc/agentsexternalstubs/controllers/AgentSuspensionController.scala

```diff
@@ -26,16 +26,16 @@ class AgentSuspensionController @Inject()(
           case None => Future successful notFound("USER_NOT_FOUND")
           case Some(user) =>
             user.suspendedServices match {
-              case None     => Future successful Ok(Json.toJson(AgentSuspensionResponse("NotSuspended")))
-              case Some(ss) => Future successful Ok(Json.toJson(AgentSuspensionResponse("Suspended", Some(ss))))
+              case None     => Future successful Ok(Json.toJson(SuspendedServices(Set.empty)))
+              case Some(ss) => Future successful Ok(Json.toJson(SuspendedServices(ss)))
```

Comment:
minor: instead of flatMap'ing and returning Future successful in each case , just do a map, 
but no worries